{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Out of support scope",
      "package_name": "sox",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Out of support scope",
      "package_name": "sox",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    }
  ],
  "threat_severity": "Low",
  "public_date": "2023-05-05T00:00:00Z",
  "bugzilla": {
    "description": "sox: floating point exception in src/voc.c",
    "id": "2212282",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2212282"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "6.2",
    "cvss3_scoring_vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-1077",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2023-32627",
  "document_distribution": "",
  "details": [
    "A floating point exception vulnerability was found in sox, in the read_samples function at sox/src/voc.c:334:18. This flaw can lead to a denial of service.",
    "A floating point exception vulnerability was found in sox, in the read_samples function at sox/src/voc.c:334:18. This flaw can lead to a denial of service."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2023-32627\nhttps://nvd.nist.gov/vuln/detail/CVE-2023-32627"
  ]
}